src/pages/Home/Home.jsx

```diff
@@ -0,0 +1,126 @@
+import "./Home.css";
+import { useState, useEffect } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+import { Navbar } from "../../components/Navbar/Navbar";
+import { Category } from "../../components/Category/Category";
+
+const Home=()=>{
+
+const [categories, setCategories] = useState([]);
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/categories");
```

Comment:
Suggestion: You can make services for each api call.

Reply:
i will in my upcoming PRs